examples/searching/Linear Search/Go/main.go

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
See https://github.com/johnfraserss/ICS4U/issues/37 for planned update regarding the searching algorithms.